bigip/resource_bigip_ltm_snat.go

```diff
@@ -209,7 +216,7 @@ func SnatToData(p *bigip.Snat, d *schema.ResourceData) error {
 	d.Set("translation", p.Translation)
 	d.Set("snatpool", p.Snatpool)
 	d.Set("vlansdisabled", p.VlansDisabled)
-
+	d.Set("vlans", p.Vlans)
```

Comment:
Since this is an aggregate resource type, you need to error check the set.

https://www.terraform.io/docs/extend/best-practices/detecting-drift.html#error-checking-aggregate-types